modin/core/storage_formats/pandas/parsers.py

```diff
@@ -152,7 +152,12 @@ def generic_parse(fname, **kwargs):
             return callback(fname, **kwargs)
 
         # pop "compression" from kwargs because bio is uncompressed
-        with OpenFile(fname, "rb", kwargs.pop("compression", "infer")) as bio:
+        with OpenFile(
+            fname,
+            "rb",
+            kwargs.pop("compression", "infer"),
+            **(kwargs.pop("storage_options", None) or {}),
```

Comment:
I wonder why is this done in refactoring PR...

Reply:
It's a issue after rebase. It's [already done](https://github.com/modin-project/modin/blob/8d67cd3a467d1eb2262898f6f5e2cf159c27d7b0/modin/core/storage_formats/pandas/parsers.py#L238) for CSV and FWF parsers in master branch.